### ISSUE TITLE
Drawer Accessibility Violation - WCAG contrast ratio threshold

### ISSUE THREAD

**Issue by ZhuoyuJin** [Sage/carbon#4728]
### Current behaviour

`with-footer` and `side-view-navigation` stories in Drawer component have accessibility violations - "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds".

### Expected behaviour

No accessibility violations

### CodeSandbox or Storybook URL

https://carbon.sbc-ui.com/?path=/story/drawer--with-footer

### Carbon Version

latest

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
@ZhuoyuJin I think this is a false positive in AXE, when the component is animating AXE can't test the contrast because the element isn't "visible", and so it fails be default. This is the same issue as https://github.com/Sage/carbon/issues/4451
